sdk/communication/azure-communication-networktraversal/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.0-beta.3 (2024-04-11)
+## 1.1.0 (2024-04-11)
```

Comment:
The version should be updated in `pom.xml` too and to do that you should first update `version_client.txt` and then run update_version.py script. See [this documentation](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#tooling-version-files-and-marker-tags) for more details.

Reply:
Updated